### DISCUSSION TITLE
Optionally suppress paging with HTML Output

### DISCUSSION THREAD

**Discussion by dmenne** [quarto-dev/quarto-cli#3288] [Feature Requests]
bookdown classic produces a continuous HTML-file with a .yml to define the chapters. Quarto be default adds paging.

Feature request: `paging: false` option to suppress paging

Use case: I want one file (with embed-resources) to forward to collaborators: "This is the report" instead of "To read the report, unzip all files, then click on index.html".

See also #3267

**Comment by mcanouil** [comment]
I did not realise at first, but `html` format has no "paging" ...
Are you using the book project format? A book is not supposed to be a long HTML document, for this there is the `html` format.

Also, what I understand you want is: as for book project with the `book.chapters` option, have a `include` or `qmd-files` general option to list a set of Quarto documents to be included sequentially in the body of a main document without having to use N include shortcode.
This new YAML option could be used for any formats.
